src/main/resources/public/ts/controllers/controller.ts

```diff
@@ -911,5 +911,11 @@ export const SupportController: Controller = ng.controller('SupportController',
 			}
 		};
 
+		$scope.disableCreateTicket = () => {
+			if ($scope.ticket.category == null || !$scope.ticket.subject || !$scope.ticket.description){
+				return true
+			}
+		}
```

Comment:
- Il manque le typage de retour
- Même si c'est fonctionnel en JS, il est préférable d'avoir un return false a la fin
- Il n'y a pas besoin de faire des return True ou False.
- Pour éviter les NPE il faut rajouter une condition sur ticket

```suggestion
		$scope.disableCreateTicket = () : boolean => {
		        return !$scope.ticket || $scope.ticket.category == null || !$scope.ticket.subject || !$scope.ticket.description
		}
```

Reply:
Et j'ajouterai peut être même que le **$scope.ticket.category == null** est simplifiable par un **!$scope.ticket.category**, mais faut vérifier pour le coup. La différence ce fera sur un cas ou ta variable vaut "" par exemple :
- **"" == null** te renvoie false
- **!"" te renvoie** true